doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -227,6 +227,16 @@ Bluetooth libraries and services
 
 |no_changes_yet_note|
 
+Bluetooth mesh profile
+----------------------
+
+* :ref:`bt_mesh_dk_prov` module: Changed the UUID generation to prevent trailing zeros in the UUID.
+
+  **Migration notes:** To retain the legacy generation of UUID, enable the option ``CONFIG_BT_MESH_DK_LEGACY_UUID_GEN``.
+
+
+See `Bluetooth mesh samples`_ for the list of changes for the Bluetooth mesh samples.
+
```

Comment:
```suggestion
* :ref:`bt_mesh` library:

  * :ref:`bt_mesh_dk_prov` module: Changed the UUID generation to prevent trailing zeros in the UUID.

    **Migration notes:** To retain the legacy generation of UUID, enable the option ``CONFIG_BT_MESH_DK_LEGACY_UUID_GEN``.

See `Bluetooth mesh samples`_ for the list of changes for the Bluetooth mesh samples.

```

Reply:
Hope I got the indentation right, it's not always easy to see in the comment field. You might want to double-check before pushing if you are copy-pasting :)